code/modules/jobs/job_types/wasteland.dm

```diff
@@ -268,7 +268,7 @@ Raider
 		/obj/item/ammo_box/magazine/m10mm/adv/simple=2,
 		/obj/item/melee/onehanded/knife/bayonet = 1,
 		/obj/item/storage/box/ration/ranger_breakfast = 1,
-		/obj/item/book/granter/crafting_recipe/blueprint/m1carbine = 1)
+		/obj/item/book/granter/crafting_recipe/blueprint/r82 = 1)
```

Comment:
Are they supposed to get an r82 blueprint?

Reply:
FUCK FUCK FUCK FUCK FUCK FUCK FUCK

Reply:
BEAR FIX OR I MERGE WITH YOUR FACE

Reply:
fixed